internal/kibana/reconciler_test.go

```diff
@@ -418,6 +419,42 @@ func TestDeploymentDifferentWithProxyEnvVar(t *testing.T) {
 	}
 }
 
+func TestKibanaSecurityContext(t *testing.T) {
+	clusterRequest := &KibanaRequest{
+		client: nil,
+		cluster: &kibana.Kibana{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: "test-namespace",
+			},
+			Spec: kibana.KibanaSpec{},
+		},
+	}
+
+	podSpec := newKibanaPodSpec(clusterRequest, "test-app-name", nil, nil, "")
+
+	if !*podSpec.SecurityContext.RunAsNonRoot {
+		t.Errorf("SecurityContext.RunAsNonRoot: expected true, actual %v", *podSpec.SecurityContext.RunAsNonRoot)
+	}
+
+	expected := v1.SecurityContext{
+		ReadOnlyRootFilesystem:   pointer.BoolPtr(true),
+		AllowPrivilegeEscalation: pointer.Bool(false),
+		Capabilities: &v1.Capabilities{
+			Drop: []v1.Capability{"ALL"},
+		},
+	}
+
+	for _, c := range podSpec.Containers {
+		if reflect.DeepEqual(c.SecurityContext, expected) {
+			t.Errorf("Container.SecurityContext: expected %+v, actual %v", expected, c.SecurityContext)
+		}
+	}
+}
+
+/**********************/
+/** Helper functions **/
+/**********************/
+
```

Comment:
We can omit such separators here as test function per convention named  `Test..`